pom.xml

```diff
@@ -12,7 +12,7 @@
   <version>1.36-SNAPSHOT</version>
 
   <properties>
-    <java.level>3</java.level>
+    <java.level>8</java.level>
```

Comment:
The purpose of such ancient dependency is to print a human-readable warning when somebody tries to start Jenkins with wrong Java. It would be nice to keep it at Java 6 at this point.

Which Java version do you need for `setContextClassLoader()`?